src/pages/Commits.tsx

```diff
@@ -118,7 +119,10 @@ export const Commits = () => {
   const { badges, handleOnRemove, handleClearAll } = useFilterBadgeQueryParams(
     queryParamsToDisplay
   );
-  const onSubmitTupleSelect = ({ category }: { category: string }) => {
+  const onSubmit = useUpsertQueryParams();
+
+  const onSubmitTupleSelect = ({ category, value }) => {
+    onSubmit({ category, value });
```

Comment:
I wonder if it would make more sense to call `useUpsertQueryParams` inside of the `TupleSelect` component since that's the only place where it's used on this page. 🤔 At least, that's the pattern that the `HistoryTableTestSearch` uses